third_party/com_github_googleapis_google_cloud_cpp.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/google/cloud/storage/BUILD b/google/cloud/storage/BUILD
+index d48b297..a7dd6a6 100644
+--- a/google/cloud/storage/BUILD
++++ b/google/cloud/storage/BUILD
+@@ -76,7 +76,7 @@ cc_library(
+         "//google/cloud:google_cloud_cpp_common",
```

Comment:
Let's remove this line from the diff. It causes errors internally and is not really needed

Reply:
Ok. so we will keep the google-cloud-cpp 1.14 and remove this line ?

Reply:
We need to remove this line in the patch if we keep google-cloud-cpp 1.14. But if we keep it we also need to move all users of the other google-cloud-cpp to the new library. Let's make one PR that just handles the changes to `third_party` and `workspace.bzl` so that we can test all downstream projects that they are ok with upgrading to `google-cloud-cpp` 1.14. Then, we rebase this one on top of that one.